ui/perfherder/alerts/StatusDropdown.jsx

```diff
@@ -63,6 +63,7 @@ export default class StatusDropdown extends React.Component {
     }
 
     const templateArgs = {
+      revisionId: alertSummary.revision,
```

Comment:
Where is this used?

Reply:
First sentence of the description, I just updated the PR.

Reply:
Why not simply `revision`? `revisionId` could falsely hint this as a database/implementation detail...